README.md

```diff
@@ -115,4 +115,4 @@ for examples of how to start using databases together with SQLAlchemy core expre
 [quart]: https://gitlab.com/pgjones/quart
 [aiohttp]: https://github.com/aio-libs/aiohttp
 [tornado]: https://github.com/tornadoweb/tornado
-[fastapi]: https://github.com/tiangolo/fastapi
\ No newline at end of file
+[fastapi]: https://github.com/tiangolo/fastapi
```

Comment:
What happened here? Was a newline inserted or removed?

Reply:
This just looks like a trailing newline which seems fine